schemas/styles/csl-data.rnc

```diff
@@ -1,14 +1,20 @@
+namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
 namespace dc = "http://purl.org/dc/elements/1.1/"
 
 dc:title [ "Citation Style Language Data" ]
 dc:creator [ "Bruce D'Arcus" ]
-dc:rights [ "Copyright 2009-2018 Citation Style Language and contributors" ]
```

Comment:
Why are there all these formatting/whitespace changes?

Reply:
It's what trang is changing when I run it.

Reply:
Interesting. I guess the question is how the repo ended up with the funky formatting.

Oh well, not germane to the PR.